src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -69,7 +96,11 @@ const OnDemandTvPage = ({ pageData }) => {
         description={shortSynopsis}
         openGraphType="website"
       />
-      <LinkedData type="WebPage" seoTitle={headline} />
+      <LinkedData
+        type="WebPage"
+        seoTitle={headline}
+        entities={episodeIsAvailable ? [videoLinkedData] : []}
```

Comment:
sorry, just realised another thing. Doesn't need to be done but to prevent creating an object called `videoLinkedData` calls a couple of date functions you could just pop the object right in there like


```jsx
entities={
  episodeIsAvailable
    ? [
        {
          '@type': 'VideoObject',
          name: promoBrandTitle,
          description: shortSynopsis,
          thumbnailUrl: thumbnailImageUrl,
          duration: durationISO8601,
          uploadDate: new Date(releaseDateTimeStamp).toISOString(),
          embedURL: embedUrl,
        },
      ]
    : []
}
```

you could argue this might be less clear or something though